readme.txt

```diff
@@ -52,7 +52,7 @@ The positions are mostly self-explanatory. When you check the reference against
 == Installation ==
 
 The Ads for Genesis plugin extends the free Advanced Ads plugin, a simple and powerful ad management solution for WordPress. Before using this plugin download, install and activate Advanced Ads for free from https://wordpress.org/plugins/advanced-ads/.
-You can use Advanced Ads along any other ad management plugin and don’t need to switch completely.
+You can use Advanced Ads along any other ad management plugins like Ad Inserter or WP QUADS and don’t need to switch completely.
```

Comment:
First, I wanted to say No to using "WP QUADS", but there are very few results for their brand name in the search, so why not